package.json

```diff
@@ -36,6 +36,7 @@
     "zustand": "^4.5.0"
   },
   "devDependencies": {
+    "@types/cypress": "^1.1.6",
```

Comment:
Do we really need this? https://www.npmjs.com/package/@types/cypress says it's deprecated and included in cypress itself.